modules/tinymce/src/plugins/wordcount/test/ts/browser/PluginTest.ts

```diff
@@ -63,4 +63,19 @@ describe('browser.tinymce.plugins.wordcount.PluginTest', () => {
     editor.setContent('<p>Soft hy&shy;phen</p>');
     await pWaitForWordcount(2);
   });
+
+  it('TINY-7484: Does not treat zwsp character as splitting a word', async () => {
```

Comment:
Can we please add a test for character counting here, as it should still be included there?

Reply:
and I just noticed nothing in this tests characters :cry: we can probably ignore that now and log a tech debt task to add tests for that.